assignments/resume-bootstrap/resume.html

```diff
@@ -0,0 +1,229 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="bootstrap.min.css">
+    <link rel="stylesheet" href="resume.css">
+</head>
+
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <header class="header">
+                <div class="col-xs-12 col-md-5 col-sm-5" id="headerleft">
+                    <div id="headerdot"></div>
+                    <h1>Reem Gamal </h1>
+                    <p id="hp">Junior Software Engineer </p>
+                </div>
+                <div class="col-xs-12 col-md-2 col-sm-2" id="headermiddel">
+                    <img src="resume img/reem.png">
+                </div>
+                <div class="col-xs-12 col-md-5 col-sm-5" id="headerright">
+                    <ul id="hul">
+                        <li>
+                            <img src="resume img/mail.png">
+                            <p class="headerrightp">reem_gamal2008@ yahoo.com</p>
+
+                        </li>
+                        <li>
+                            <img src="resume img/phone.png">
+                            <p class="headerrightp">+2-0112­774­8360</p>
+
+                        </li>
+                        <li>
+                            <img src="resume img/earth.png">
+                            <p class="headerrightp">Cairo, ­Egypt</p>
+
+                        </li>
+                        <li>
+                            <img src="resume img/git.png">
+                            <address>
+                                <a href="http://www.github.com/ReemG">www.github.com/ReemG</a>
+                            </address>
+
+                        </li>
+                    </ul>
+                </div>
+            </header>
+        </div>
+
+        <div class="row">
+            <main class="main-wrap">
+                <div class="col-xs-12 col-md-6 col-sm-6" id="leftcontent">
+                    <article>
+                        <h2>Work Experience </h2>
+                        <div class="maindot"></div>
+                        <address>
+                            <p class="pdate"> 11/2015-02/2016</p>
+                            <p class="pplace">Cairo,Egypt</p>
+                        </address>
+                        <h3> Call Center </h3>
+                        <h4>Xceed </h4>
+                        <br>
+                        <div class="maindot"></div>
+                        <address>
+                            <p class="pdate"> 02/2016-04/2016</p>
+                            <p class="pplace">Cairo,Egypt</p>
+                        </address>
+                        <h3> Software Developer </h3>
+                        <h4> Arab Programers Company </h4>
+                        <ul id="workul">
+                            <li>Develop an desktop applications using .Net</li>
+                        </ul>
+                    </article>
+                    <article>
```

Comment:
I would use `<section>` for Experience, Education, skills and use `<article>` inside these sections.